torchgeo/trainers/classification.py

```diff
@@ -67,7 +72,8 @@ def config_model(self) -> None:
             )
         else:
             raise ValueError(
-                f"Model type '{classification_model}' is not a valid timm model."
+                f"Model type '{classification_model}' is not a valid timm model. "
+                f"Valid model types are {valid_models}."
```

Comment:
This is a very long list. I don't think we want to print this.

Reply:
Fair point, I added a code block in the docstring instead to see available models. Is that okay?